docs/buildah-import.md

```diff
@@ -0,0 +1,72 @@
+% BUILDAH(1) Buildah User Manuals
+% Buildah Community
+% JUNE 2017
```

Comment:
Looks like only bulidah-import.md and buildah-export.md have these three lines.  Should we add them to the others?  Not for this PR, but for a follow on.